tests/bsim/run_parallel.sh

```diff
@@ -51,7 +52,8 @@ elif [ -n "${TESTS_LIST}" ]; then
 	all_cases=${TESTS_LIST}
 else
 	SEARCH_PATH="${SEARCH_PATH:-.}"
-	all_cases=`find ${SEARCH_PATH} -name "*.sh" | grep -Ev "${sh_filter}"`
+	all_cases=`find ${SEARCH_PATH} -name "*.sh" | grep -Ev "${sh_filter}"` \
+	          | (echo "No tests found")
```

Comment:
Not sure what is right?! (am not a bash script expert)

```suggestion
	all_cases=`find ${SEARCH_PATH} -name "*.sh" | grep -Ev "${sh_filter}" | echo "No tests found"`
```